geoconvert/data/countries.py

```diff
@@ -24,7 +24,14 @@
     # Make sure we never mistake it for JE
     "new jersey": "US",
     # Make sure we never mistake it for GN
-    "papua new guinea": "PG"
+    "aquatorialguinea": "GQ", # de
+    "equatorial guinea": "GQ", # en
+    "guinea bissau": "GW", # en / de
+    "guinea ecuatorial": "GQ", # es
+    "pap new guinea": "PG", # en
+    "papua neuguinea": "PG", # de
+    "papua new guinea": "PG", # en
+    "papua nueva guinea": "PG", # es
```

Comment:
Do not forget to add the French and Portuguese country names here as well.

Reply:
Actually, there is no need for that (because we have `_full_name_to_country_code_for_lang` for that). But we might want to add tests about these languages (for both Guinea and South Sudan), just to make sure we do not break it.